### ISSUE TITLE
jruby/bin/jruby permission denied in bin/game_machine.sh

### ISSUE THREAD

**Issue by SacWebDeveloper** [gamemachine/gamemachine#11]
Hi Chris,

I extracted the server.zip to ````/opt/game_machine````, made the script ````bin/game_machine.sh```` executable, and upon trying attempting to run the ````bin/game_machine.sh build clean```` command, it errors out with  ````bin/game_machine.sh: line 13: jruby/bin/jruby: Permission denied````. Even when running the script with sudo, I get the same. What I am missing? The documentation doesn't say. Also, is there anyway for others to contribute to the documentation when we run into issues like this?

Thanks,

Kyle

**Comment by gamemachine** [comment]
Unfortunately no way to contribute to docs.

I'll get a new server.zip up this weekend.  I can't remember how I made it but the permissions should not be messed up like that. You can probably get it to work by just doing a chmod 755 on the files it complains about.  There aren't many that need to be executable.

**Comment by SacWebDeveloper** [comment]
I'll try this later and get back to you on that.

**Comment by SacWebDeveloper** [comment]
So I got the ````bin/game_machine.sh build clean```` to output BUILD SUCCESSFUL, but it took running the command twice after doing a chmod to the jruby executable.

````sudo chmod +x jruby/bin/jruby````

Unfortunately, I lost the error message of the first run, but it did say BUILD FAILED.

I noticed that when you download jruby from the official site, the ````bin```` directory already has the executable permission enabled. Perhaps just use the permissions that are used with the official distribution.
